plugin/trino-exasol/src/main/java/io/trino/plugin/exasol/ExasolClient.java

```diff
@@ -206,6 +212,13 @@ public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connect
         if (mapping.isPresent()) {
             return mapping;
         }
+        if (isHashTypeMapping(typeHandle)) {
+            Boolean useToCharMapping = session.getProperty(MAP_HASHTYPE_AS_CHAR, Boolean.class);
```

Comment:
Add a helper method to ExasolSessionProperties, and make `MAP_HASHTYPE_AS_CHAR` private.